package/wireguard-rm2/package

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(wireguard-rm2)
+pkgdesc="Fast, modern, secure VPN tunnel"
+url=https://www.wireguard.com
+pkgver=1.0.20201221
+timestamp=2020-12-21T11:54Z
+section=utils
+maintainer="Jonah Weissman <jonahrweissman+toltec@gmail.com>"
```

Comment:
https://github.com/toltec-dev/toltec/blob/testing/docs/package.md#maintainer You okay with being the one in charge of reviewing alongside one of us with merge power? Or would you prefer for one of us to own that responsibility for now?

Reply:
Yeah! I'm okay with reviewing, if that's okay with you. Now that I've spent all this time getting it to work in the first place, it would feel good to be able to continue to use my skills to help out.

Reply:
If you aren't already I'd highly recommend joining the community discord and poking Ameb for the developer role. We do a lot of talking in there.